site/gatsby-site/migrations/2025.04.25T16.31.55.migrate-incident-88.js

```diff
@@ -0,0 +1,115 @@
+/** @type {import('umzug').MigrationFn<any>} */
+exports.up = async ({ context: { client } }) => {
+  try {
+    await client.connect();
+    const db = client.db('aiidprod');
+
+    const incidents = db.collection('incidents');
+
+    const reports = db.collection('reports');
+
+    const duplicates = db.collection('duplicates');
+
+    const classifications = db.collection('classifications');
+
+    // Find incident 88
+    const incident88 = await incidents.findOne({ incident_id: 88 });
+
+    if (!incident88) {
+      console.error('Incident 88 not found. Migration cannot proceed.');
+      return;
+    }
+
+    // Get reports associated with incident 88
+    const incident88Reports = incident88.reports;
+
+    // Find the last incident to determine the new incident_id
+    const lastIncident = await incidents.find({}).sort({ incident_id: -1 }).limit(1).next();
+
+    // Calculate new incident_id
+    const incident_id = lastIncident ? lastIncident.incident_id + 1 : 1;
```

Comment:
I think we should throw an error and cancel the migration if the `lastIncident` is not defined instead of set the `incident_id` as number `1`